docs/howto/garbage-collection.md

```diff
@@ -286,9 +286,7 @@ using the feature you may want to first gain confidence in the decisions GC make
 * Backup: copy expired objects from your repository's storage namespace to an external location before running GC in [sweep only mode](#sweep-only-mode).  
 * Restore: copy objects that were hard-deleted by GC from an external location you used for saving your backup into your repository's storage namespace.
 
-### Using rclone
-
-Follow [rclone documentation](https://rclone.org/docs/) to configure remote access to the underlying storage used by lakeFS.
+Follow [**rclone** documentation](https://rclone.org/docs/) to configure remote access to the underlying storage used by lakeFS.
```

Comment:
Related?

Reply:
yes, I. only kept a single option that's left therefore removed the title that says rclone or distcp

Reply:
ok, I removed it. thanks!